sequence_processing_pipeline/tests/test_Pipeline.py

```diff
@@ -171,7 +170,7 @@ def test_creation(self):
                      self.good_run_id,
                      self.bad_assay_type_path, None,
                      self.output_file_path,
-                     self.qiita_id, None)
+                     self.qiita_id, 'metagenomic', None)
```

Comment:
I'll just note here, not on the below, but it would be ideal for this to be the static variable suggested in an earlier comment